homeassistant/components/peco/sensor.py

```diff
@@ -120,4 +122,5 @@ def __init__(
     @property
     def native_value(self) -> int:
         """Return the value of the sensor."""
-        return self.coordinator.data[self._key]
+        data: int = self.coordinator.data[self._key]
+        return data
```

Comment:
This is not entirely true. In `async_update_data()` you are setting it to a `str`
```py
if data["percent_customers_out"] < 5:
    data["percent_customers_out"] = "Less than 5%"
```